database/sqlite/sqlite_aclk_alert.c

```diff
@@ -527,14 +542,14 @@ int aclk_push_alert_config_event(struct aclk_database_worker_config *wc, struct
     }
 
     if (likely(p_alarm_config.cfg_hash)) {
-        log_access("OG [%s (%s)]: Sent alert config %s.", wc->node_id, wc->host ? wc->host->hostname : "N/A", config_hash);
+        log_access("ACLK RES [%s (%s)]: Sent alert config %s.", wc->node_id, wc->host ? wc->host->hostname : "N/A", config_hash);
```

Comment:
minor comment - Should we have those prefixes (ACLK_STA/RES/REQ) as defines or define macros like `log_access_sta` etc.

should help prevent the executable from having those strings repeated there million times?

Reply:
For sure we can revise with a future PR